railties/lib/rails/test_help.rb

```diff
@@ -20,30 +20,32 @@
     exit 1
   end
 
-  module ActiveSupport
-    class TestCase
-      include ActiveRecord::TestDatabases
-      include ActiveRecord::TestFixtures
-      self.fixture_path = "#{Rails.root}/test/fixtures/"
-      self.file_fixture_path = fixture_path + "files"
-    end
+  ActiveSupport.on_load(:active_support_test_case) do
+    include ActiveRecord::TestDatabases
+    include ActiveRecord::TestFixtures
+
+    self.fixture_path = "#{Rails.root}/test/fixtures/"
+    self.file_fixture_path = fixture_path + "files"
   end
 
-  ActionDispatch::IntegrationTest.fixture_path = ActiveSupport::TestCase.fixture_path
+  ActiveSupport.on_load(:action_dispatch_integration_test) do
+    self.fixture_path = ActiveSupport::TestCase.fixture_path
+  end
 end
 
 # :enddoc:
 
-class ActionController::TestCase
+module Routing
```

Comment:
This could clash with a top-level constant named `Routing` in the user's application.